ats/src/atsMachines/slurmProcessorScheduled.py

```diff
@@ -286,25 +298,39 @@ def calculateCommandList(self, test):
         #
         # --exclusive is the default, but it can be switched via the --share ats argument
         #
+        ex_or_sh = '--comment="noexclusive"'
+        exact = '--comment="noexact"'
         if self.exclusive == True:
-            ex_or_sh = "--exclusive"
-        else:
-            ex_or_sh = "--share"
+            if SlurmProcessorScheduled.slurm_version_int >= 21100:
+                ex_or_sh = "--overlap"      
+                # ex_or_sh = "--exclusive=user" 
+                # exact = "--exact"
+                # exact = "--cpu_bind=cores"
+            else:
+                ex_or_sh = "--exclusive"
+                # ex_or_sh = "--exclusive=user"
 
         temp_uname = os.uname()
         host = temp_uname[1]
 
-        the_mpi_type='-v'
+        the_mpi_type='--comment="nompitype"'
         if host.startswith('rznevaxxx'):
             the_mpi_type='--mpi=pmi2'
 
+        unbuffered='--comment="nounbuffered"'
+        if configuration.options.unbuffered:
+            unbuffered='--unbuffered'
 
         if MY_SYS_TYPE.startswith('toss'):
             # none means to not spcify mpibind at all
             if self.mpibind == "none":
                 mpibind = "--epilog=none"
             else:
-                mpibind = "--mpibind=%s" % self.mpibind
+                if SlurmProcessorScheduled.slurm_version_int >= 21100:
```

Comment:
The "if" and "else" statements in this "else" block have the same effect. Revert back to just...
```
else:
    mpibind = "--mpibind=%s" % self.mpibind
```
??

Reply:
Hi David,  I agree that if this is the final solution, we should remove the redundancy.  However, I am still testing options with the slurm update on alastor.  Can we leave this in for a bit so I can explore?

Reply:
Of course! I'm just seeing this as a confusing update to the code in the main branch. Also I'm assuming that
`# mpibind = "--mpibind=%s" % 'on'` should be uncommented?

Reply:
No, that is commented out on purpose.  Still exploring that.

Reply:
Let's live with the confusion for a bit, as the slurm update has added confusion we are trying to adjust to.

Reply:
Just to be sure, this is the proposed changed.
```
if SlurmProcessorScheduled.slurm_version_int >= 21100:
    mpibind = "--mpibind=%s" % self.mpibind
else:
    mpibind = "--mpibind=%s" % self.mpibind
```
Both outcomes are identical... I don't understand what this update does 🤷‍♂️

Reply:
Yes, I know it looks weird.  But I want to work on that a bit.  I am researching different mpibind options which may be useful for Slurm version > 21100, but am not ready to commit yet.  I don't want to lose the location where I am doing this experimentation.

Reply:
Got it! Good to know.